mqclient/abstract_backend_tests/integrate_queue.py

```diff
@@ -89,16 +89,18 @@ async def test_12(self, queue_name: str) -> None:
         """Failure-test one pub, two subs (one subscribed to wrong queue)."""
         all_recvd: List[Any] = []
 
-        async with Queue(self.backend, name=queue_name).sender() as s:
-            await s.send(DATA_LIST[0])
+        async with Queue(self.backend, name=queue_name).open_pub() as p:
+            await p.send(DATA_LIST[0])
             _log_send(DATA_LIST[0])
 
         with pytest.raises(Exception) as excinfo:
-            async with Queue(self.backend, name=f"{queue_name}-fail").recv_one() as d:
+            async with Queue(
+                self.backend, name=f"{queue_name}-fail"
+            ).open_sub_one() as d:
                 all_recvd.append(_log_recv(d))
             assert "No message available" in str(excinfo.value)
```

Comment:
Just a question: does this assert actually get called?  Or does it need to be un-indented?

Reply:
that indentation is nasty, but the code flow is correct. I'll make it more clear